recipes/formidable/README.md

```diff
@@ -0,0 +1,47 @@
+## formidable instance
+
+Now you can send your own `formidable.IncomingForm` instance via `options.IncomingForm`. It means you can do something beyond default behavior, such as handling image upload.
+
+Below is the simple demo of custom `formidable.IncomingForm` instance and event handling. Check [formidable](https://github.com/felixge/node-formidable) for more info.
+
+```js
+'use strict'
+
+var app = require('koa')()
+var path = require('path')
+var IncomingForm = require('formidable')
+var body = require('../../index')
```

Comment:
make more sense in the readme to be `require('koa-better-body')` to allow users to just copy and paste.